src/XrdSciTokens/XrdSciTokensAccess.cc

```diff
@@ -964,7 +974,16 @@ class XrdAccSciTokens : public XrdAccAuthorize, public XrdSciTokensHelper,
                     xrd_rules.emplace_back(AOP_Update, path);
                     xrd_rules.emplace_back(AOP_Chmod, path);
                     xrd_rules.emplace_back(AOP_Stat, path);
+                    xrd_rules.emplace_back(AOP_Poll, path);
                     xrd_rules.emplace_back(AOP_Delete, path);
+                } else if (!strcmp(acl_authz, "storage.stage")) {
+                    xrd_rules.emplace_back(AOP_Stage, path);
+                    xrd_rules.emplace_back(AOP_Poll, path);
+                    xrd_rules.emplace_back(AOP_Read, path); // May be removed
+                    xrd_rules.emplace_back(AOP_Readdir, path); // May be removed
+                    xrd_rules.emplace_back(AOP_Stat, path);
```

Comment:
Having stage does not imply anything other than stage and (I think) poll. So, one should not add read, readdir, or stat. Note here you do seem to concur that stage implies poll, so my previous comment on privilege bits is correct.